.gitpod.yml

```diff
@@ -1,8 +1,8 @@
 image:
   file: Dockerfile
 tasks:
-  - init: touch .env
-    command: DEBUG_ENV=1 SECRET_KEY=dskaj343 CPPFLAGS="$(pg_config --cppflags)" LDFLAGS="$(pg_config --ldflags)" bazel build :manage --watchfs --spawn_strategy=standalone --copt --aspects=@bazel_tools//tools/python:srcs_version.bzl%find_requirements --verbose_failures=true --show_timestamps=true --python_version=PY3 --build_python_zip --sandbox_debug --color=yes --curses=yes --jobs=2000 --loading_phase_threads=HOST_CPUS --action_env=LDFLAGS="$(pg_config --ldflags)" --action_env=CPPFLAGS="$(pg_config --cppflags)" --action_env=DEBUG_ENV --action_env=SECRET_KEY
+  - init: sudo docker-up
```

Comment:
What is `docker-up` ?

Reply:
preview feature on gitpod that enables docker compose to run in root